infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -286,7 +286,7 @@ private void handleException(Throwable ex, ChannelHandlerContext ctx, RequestMsg
     private void handleWrite(RequestMsg req, ChannelHandlerContext ctx, IServerRouter router) {
         LogData logData = getLogData(req.getPayload().getWriteLogRequest().getLogData());
 
-        log.debug("handleWrite: type: {}, address: {}, streams: {}",
+        log.debug("handleWrite: {}, at: {}, ids: {}",
```

Comment:
I honestly prefer the previous one better.

Reply:
Yeah, the issue here is that it's the most printed line in our entire log (maybe > 90% of lines are attributed to it) and nobody wants to remove it because apparently, it helps with debugging, so I decided to reduce it at least a little bit